README.md

```diff
@@ -60,6 +60,57 @@ export MARQUEZ_TIMEOUT_MS=5000
 
 ```
 
+### Common Calls and Examples
+There are two main ways to use the client:
+1. Set the namespace
+2. Supply the namespace for calls that require it
+
+#### Instantiating the client
+```mc = Client()```
+
+#### Set a Namespace for the client to use
+```mc.set_namespace(existing_namespace)```
+
+#### Create a Namespace
+``` 
+mc.create_namespace(namespace_name, 
```

Comment:
`namespace_name` isn't defined in your example -- I think putting an example string into the call is fine, just like you're doing with owner name.

Reply:
Makes sense. Will remove undefined vars.